cloudformation/xtdb_ecs.yml

```diff
@@ -0,0 +1,239 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Parameters:
+  SecurityGroupId:
+    Description: Group ID of the security group to be used by the ECS nodes
+    Type: AWS::EC2::SecurityGroup::Id
+
+  PublicSubnets:
+    Description: List of public subnets to host the load balancer on
+    Type: List<AWS::EC2::Subnet::Id>
+
+  TargetGroupArn:
+    Description: ARN of the Target Group created for the nodes
+    Type: String
+    
+  LoadBalancerArn:
+    Description: ARN of the Application Load Balancer created for the nodes
+    Type: String
+
+  S3BucketName:
+    Description: Name of the S3 bucket to use as the XTDB object store
+    Type: String
+  
+  SNSTopicArn:
+    Description: The ARN of the SNS Topic which the S3 sends notifications to
+    Type: String
+    
+  S3AccessPolicyArn:
+    Description: ARN of the managed policy offering access to all the S3 permissions necessary for the object store
+    Type: String
+  
+  MSKBootstrapServers:
+    Description: Comma separated list containing all Kafka bootstrap server URLs from MSK
+    Type: String
+
+  MSKAccessPolicyArn:
+    Description: ARN of the managed policy offering access to all the MSK permissions
+    Type: String
+
+  ClusterName:
+    Description: Name of the desired ECS cluster.
+    Type: String
+    Default: xtdb-cluster
+
+  EC2InstanceType:
+    Description: EC2 Instance Type used for ECS Service (We recommend using one that is storage optimized)
```

Comment:
worth confirming that Docker can use the instance storage, rather than just the (much smaller) root partition of the machine